zip-0400.rst

```diff
@@ -0,0 +1,174 @@
+::
+
+  ZIP: 400
+  Title: Wallet.dat format
+  Owners: Alfredo Garcia <oxarbitrage@gmail.com>
+  Status: Draft
+  Category: Wallet
+  Created: 2020-05-26
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST" and "MAY" in this document are to be interpreted as described in
+RFC 2119. [#RFC2119]_
+
+Abstract
+========
+
+This proposal defines the current zcash format for wallet persistence storage commonly known as wallet.dat.
+
+Motivation
+==========
+
+The process of saving wallet data into disk is currently unspecified. The key-values used in the current implementation are undocumented and by this structure and functionality is unknown without a deep analysis of the involved source code. This document details the schema and the mechanics of the wallet database. This is an informational document, no changes will be done to the source code.
+
+Specification
+=============
+
+Zcash stores wallet information in a Berkeley database(BDB) commonly known as wallet.dat. 
```

Comment:
Should add a reference.

Reply:
Added at https://github.com/zcash/zips/pull/372/commits/d5336ffecc31a25d89a89ef1cc5c483d254aabc0 . Thanks.